mappings/net/minecraft/client/gui/screen/LoadingDisplay.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_62 net/minecraft/client/gui/screen/LoadingDisplay
```

Comment:
Hmm I guess this is what it is in modern. In beta it is really just for a loading progress bar, I believe. I guess it will be more clear when the other fields eventually get mapped.